### ISSUE TITLE
Compiling FFmpeg with FDK-AAC stops at configure

### ISSUE THREAD

**Issue by BlackScreen** [rdp/ffmpeg-windows-build-helpers#561]
Nuked the whole sandbox, but it always gets back to the same error - I do not know why... This is the output:

```
Downloading (via git clone) ffmpeg_git_pre_x264_with_fdk_aac from https://github.com/FFmpeg/FFmpeg.git
Cloning into 'ffmpeg_git_pre_x264_with_fdk_aac.tmp'...
remote: Enumerating objects: 622888, done.
remote: Counting objects: 100% (287/287), done.
remote: Compressing objects: 100% (184/184), done.
remote: Total 622888 (delta 136), reused 182 (delta 103), pack-reused 622601
Receiving objects: 100% (622888/622888), 256.07 MiB | 18.81 MiB/s, done.
Resolving deltas: 100% (488535/488535), done.
Updating files: 100% (7491/7491), done.
done git cloning to ffmpeg_git_pre_x264_with_fdk_aac
doing git checkout origin/master
Note: switching to 'origin/master'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 7368e5537d avutil/cpu: Schedule deprecated functions for removal
fetched no code changes, not forcing reconfigure for that...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1430  100  1430    0     0  1396k      0 --:--:-- --:--:-- --:--:-- 1396k
applying patch frei0r_load-shared-libraries-dynamically.diff
patching file libavfilter/vf_frei0r.c
Hunk #2 succeeded at 244 (offset 5 lines).
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   443  100   443    0     0   432k      0 --:--:-- --:--:-- --:--:--  432k
applying patch ffmpeg-windres-fix.patch
patching file ffbuild/common.mak
error: patch failed: libavformat/matroskaenc.c:2330
error: libavformat/matroskaenc.c: patch does not apply
configuring ffmpeg_git_pre_x264_with_fdk_aac (/home/USER/ffmpeg/ffmpeg-windows-build-helpers/sandbox/win64/ffmpeg_git_pre_x264_with_fdk_aac) as $ PKG_CONFIG_PATH=/home/USER/ffmpeg/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/x86_64-w64-mingw32/lib/pkgconfig PATH=/home/USER/ffmpeg/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin:$PATH ./configure --pkg-config=pkg-config --pkg-config-flags=--static --extra-version=ffmpeg-windows-build-helpers --enable-version3 --disable-debug --disable-w32threads --arch=x86_64 --target-os=mingw32 --cross-prefix=/home/USER/ffmpeg/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/x86_64-w64-mingw32- --enable-libcaca --enable-gray --enable-libtesseract --enable-fontconfig --enable-gmp --enable-gnutls --enable-libass --enable-libbluray --enable-libbs2b --enable-libflite --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopus --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libvorbis --enable-libwebp --enable-libzimg --enable-libzvbi --enable-libmysofa --enable-libopenjpeg  --enable-libopenh264 --enable-liblensfun  --enable-libvmaf --enable-libsrt --enable-demuxer=dash --enable-libxml2 --enable-opengl --enable-libdav1d --enable-cuda-llvm --enable-libsvthevc --enable-libsvtav1 --enable-libsvtvp9 --enable-libaom --enable-libvpx --enable-nvenc --enable-nvdec --extra-libs=-lharfbuzz --extra-libs=-lm --extra-libs=-lpthread --extra-cflags=-DLIBTWOLAME_STATIC --extra-cflags=-DMODPLUG_STATIC --extra-cflags=-DCACA_STATIC --enable-amf --enable-libmfx --enable-gpl --enable-frei0r --enable-filter=frei0r --enable-librubberband --enable-libvidstab --enable-libx264 --enable-libx265 --enable-avisynth --enable-libaribb24 --enable-libxvid --enable-libdavs2 --enable-libxavs2 --enable-libxavs --enable-avresample --extra-cflags=-mtune=generic --extra-cflags=-O3 --enable-static --disable-shared --prefix=/home/USER/ffmpeg/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/x86_64-w64-mingw32 --enable-nonfree --enable-libfdk-aac --enable-decklink --disable-libx264
all touch files already_configured* touchname= already_configured_e445a4d773e569dc078a8c6dae1a8084-
config options --pkg-config=pkg-config --pkg-config-flags=--static --extra-version=ffmpeg-windows-build-helpers --enable-version3 --disable-debug --disable-w32threads --arch=x86_64 --target-os=mingw32 --cross-prefix=/home/USER/ffmpeg/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/x86_64-w64-mingw32- --enable-libcaca --enable-gray --enable-libtesseract --enable-fontconfig --enable-gmp --enable-gnutls --enable-libass --enable-libbluray --enable-libbs2b --enable-libflite --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopus --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libvorbis --enable-libwebp --enable-libzimg --enable-libzvbi --enable-libmysofa --enable-libopenjpeg --enable-libopenh264 --enable-liblensfun --enable-libvmaf --enable-libsrt --enable-demuxer=dash --enable-libxml2 --enable-opengl --enable-libdav1d --enable-cuda-llvm --enable-libsvthevc --enable-libsvtav1 --enable-libsvtvp9 --enable-libaom --enable-libvpx --enable-nvenc --enable-nvdec --extra-libs=-lharfbuzz --extra-libs=-lm --extra-libs=-lpthread --extra-cflags=-DLIBTWOLAME_STATIC --extra-cflags=-DMODPLUG_STATIC --extra-cflags=-DCACA_STATIC --enable-amf --enable-libmfx --enable-gpl --enable-frei0r --enable-filter=frei0r --enable-librubberband --enable-libvidstab --enable-libx264 --enable-libx265 --enable-avisynth --enable-libaribb24 --enable-libxvid --enable-libdavs2 --enable-libxavs2 --enable-libxavs --enable-avresample --extra-cflags=-mtune=generic --extra-cflags=-O3 --enable-static --disable-shared --prefix=/home/USER/ffmpeg/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/x86_64-w64-mingw32 --enable-nonfree --enable-libfdk-aac --enable-decklink --disable-libx264 ./configure
Unknown option "--enable-libsvtvp9".
See ./configure --help for available options.
failed configure ffmpeg_git_pre_x264_with_fdk_aac
```

**Comment by nanake** [comment]
> Unknown option "--enable-libsvtvp9"

dupe of #560

**Comment by rdp** [comment]
Maybe ok now?

On Wednesday, April 28, 2021, BlackScreen ***@***.***> wrote:
> Closed #561.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci4.googleusercontent.com/proxy/MSinZNgeIh8YlJsPZXI2hT0cnH29rRXR4dTSLgMqx9B9vpCuhH-a4rcAdPeBIrHPWKzYTbKCeNy5j4t645WlWms0sg3LwBmDyZcPck6KM8QWd8ebFoXEi7cXi2zczj8Ru3qb1Ndk9kaVgbqbTZRiDMWRVDIq5eMH0WVn5ZKX17u0OGMB2Szs9nYkedWd7ybGh66SVgoaS5AUk8WYUHKw1hErinLywJrTRUbqJAS-VDr9wyYQowKqh4n6WpftaQ2vmke_0d8=s0-d-e1-ft#https://github.com/notifications/beacon/AAADBUB4VKESIBRKNQSDCETTLAQHZA5CNFSM43GEIIIKYY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGPAAAAAAIVTQJP2.gif>
